Tar4/Graph.py

```diff
@@ -2,14 +2,16 @@
 # Sapir Holzman 318265097
 # Shahaf Shavit 314967324
 
+
 class Graph:
     def __init__(self):
         self.vertices = {}
 
-    def __getitem__(self, key):
-        return self.vertices[key]
+    def get_vertex(self, _id):
```

Comment:
__getitem__ is a  method in pyton, so you dont need to do `graph.get_vertex(v['id'])` you can do only `graph[v['id']]` 😄